packages/synapse-react-client/src/mocks/query/mockHomepageQueryResultData.ts

```diff
@@ -38,120 +38,135 @@ export const mockHomepageTrendingQueryResultBundle: QueryResultBundle = {
           id: '215953',
         },
         {
```

Comment:
I updated this mock data, but I don't think it's currently used in tests, since the homepage snapshot tests did not update.

It would be a bit of a pain to add tests, since we would need to add handlers for each entity in the list, so I decided against it for now.